### PR TITLE
Browser client configuration

### PR DESCRIPTION
**Pull request by switters**
It would be cool to do something like

```js
// in my user module…
var log = require('log')('user')
log.critical('user is drunk', user)

// meanwhile, in the boot module…
require('log').setAddress('https://rets.ly/log')
```

Since we won't have environment variables available. @switters good/bad?

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -31,24 +31,36 @@ else ADDR = 'http://localhost:3000'
...
   }
+
+  return getInstance(ctx)
+}
+
+
+/**
+ * Return Logger instance of same ctx or create new
+ */
+function getInstance (ctx) {
```
Comment (lvivier) [inline lib/index.js]: we can just return a new instance every call, don't need to keep track of them b/c they have no state

Comment (switters) [inline lib/index.js]: 10-4

File: lib/index.js [hunk 2]
```diff
@@ -36,19 +37,14 @@ else ADDR = 'http://localhost:3000'
...
   var send;
-  addrs = addrs || [ADDR]
-
-  return function () {
-    if(!send) {
-      send = isBrowser ? browserSend : serverSend
-    }
 
-    return new Logger(ctx, send(addrs))
+  if(!send) {
```
Comment (lvivier) [inline lib/index.js]: I think this clause isn't needed

Comment (switters) [inline lib/index.js]: Is too! ...no wait, what I meant was, "Nope, not at all."

File: lib/index.js [hunk 3]
```diff
@@ -25,9 +25,10 @@ var levels = 'debug info warn error critical alert emergency'.split(' ')
  * Default server address
  */
 
-var ADDR
-if(!isBrowser) ADDR = process.env.LOG_ADDRESS || 'tcp://localhost:57000'
-else ADDR = 'http://localhost:3000'
+if(!isBrowser) {
+  ADDR = process.env.LOG_ADDRESS || 'tcp://localhost:57000'
```
Comment (lvivier) [inline lib/index.js]: `ADDR` is global. Maybe just `setAddress(process.env.LOG_ADDRESS||'tcp://localhost:57000')`

**Comment by switters** [comment]
@lvivier - review?

File: lib/index.js [hunk 4]
```diff
@@ -58,13 +53,26 @@ exports.Logger = Logger
...
+function Logger (ctx) {
   this.ctx = ctx
 }
 
 
 /**
+ * Configure Address
+ */
+
+exports.setAddress = setAddress
```
Comment (lvivier) [inline lib/index.js]: can we move this up to the other exports for nice/obviousness?

**Comment by lvivier** [comment]
:+1:

File: lib/index.js (hunk 4 above)
Comment (switters) [inline lib/index.js]: obviating exports

**Comment by switters** [comment]
:beers:

**Pull request merged**